src/sections/dataset/dataset-files/dataset-upload-files-button/DatasetUploadFilesButton.tsx

```diff
@@ -18,7 +18,7 @@ export function DatasetUploadFilesButton() {
   }
 
   const handleClick = () => {
-    navigate(`${Route.UPLOAD_DATASET_FILES}?persistentId=${dataset.persistentId}`)
+    navigate(`${Route.UPLOAD_DATASET_FILES}?${QueryParamKey.PERSISTENT_ID}=${dataset.persistentId}`)
   }
```

Comment:
I think the handleClick() function also needs logic to add version=DRAFT, similar to EditDatasetMenu component.  When I click on the button in a draft dataset, I get "Page Not Found".

Reply:
Nice catch! I forgot about that button down there, thanks! I've fixed it.